middleware/fastcgi/fcgiclient.go

```diff
@@ -380,7 +380,10 @@ func (c *FCGIClient) Request(p map[string]string, req io.Reader) (resp *http.Res
 		if err != nil {
 			return
 		}
-		resp.Status = statusParts[1]
+		if len(statusParts) > 0 {
```

Comment:
I think Tw's referring to the line below, where you reference `statusParts[1]`. This condition guarantees that `statusParts[0]` exists but not `statusParts[1]`.